src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioWorkspaceImpl.cs

```diff
@@ -954,7 +954,7 @@ internal void UpdateDocumentContextIfContainsDocument(IVsHierarchy sharedHierarc
                 return;
             }
 
-            if (hostProject.Id != documentId.ProjectId)
+            if (hostProject?.Id != documentId.ProjectId)
```

Comment:
@dpoeschl What's the right thing here?

Reply:
Ping @dpoeschl to suggest the correct logic here.